### PR TITLE
update-readme-jc

### PR DESCRIPTION
**Pull request by joeclayallday**
**Update our readme with clear instructions for contributors.**

I'm hosting the image of the site here and referencing the URL in the readme.

Screen Shot 2023-06-23 at 10 59 47 AM

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,54 +1,73 @@
...
+**1. Install Node and Yarn**
+<br>app-ui requires node `v18.15.x` and yarn `v2+` or later.
 
+**2. Add Environment Variables**
+<br>Create a `.env` file with these variables in the root of app-ui:
 ```bash
-yarn
+VITE_AUTH_URL="https://auth.aptible.com"
+VITE_API_URL="https://api.aptible.com"
+VITE_ORIGIN="nextgen"
```
Comment (grrowe) [inline README.md:20]: I would add a note that you can also just have these env variables set with a `.env` file. 
Also, another note to tell users that these variables point to which API/Auth to use, like they could use the sbx.main ones as well.

Comment (madhuravius) [reply README.md:20]: there should be an example - https://github.com/aptible/app-ui/blob/main/.env.example

they can use that as a starting point

**Comment by joeclayallday** [comment]
Updated Mockup with revisions:
Screen-Shot-2023-06-23-at-12 51 22-PM

File: README.md (hunk 1 above)
Comment (joeclayallday) [reply README.md:20]: Added in these tweaks, thanks!

Comment (madhuravius) [reply README.md:20]: i meant they can use the example file, like:

```sh
cp .env.example .env
```

and theyll have a file ready to go they just gotta tweak

**Comment by madhuravius** [comment]
LGTM, left minor note to clarify

**Review by grrowe — APPROVED** [review]

**Pull request merged**